Finish index and about pages for mobile and desktop

s2jack opened the pull request:
Add linters
Add README
Add MIT L.icense
Create index and about html
Finish styles for mobile versions of about and index
Create main.js
Add mobile menu interaction
Add menu elements interactions
Add images and icons
Create instructor-dynamic.js to load this section dynamically
Finish styles and declare new elements for desktop versions
Fix linters

s2jack committed:
Create home page mobile version

Add linters
Create README
Add license
Create index.html
Create styles.css
Add fonts and images folder

s2jack committed:
finish mobile versions

Create about.html
create about styles css
create main.js for mobile-menu interactions

s2jack committed:
finish desktop versions

Create desktop css for about and index html
fix style issues
Add new icons and background images

s2jack committed:
Create dynamic section

Declare new js file to load whole section dynamically
Declare a database that contains all info about instructors

anagudelogu commented on README.md:
```diff
@@ -1 +1,185 @@
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> <a href="https://s2jack.github.io/codewizards.github.io/">https://s2jack.github.io/codewizards.github.io/</a>
```
- [x] Please make sure that all your links in your README are working! Currently, your live demo link is broken. If you haven't deployed your project, please consider either removing the section or having a "Coming soon" text or something similar. 🤓 🔧

anagudelogu commented on README.md:
```diff
@@ -1 +1,185 @@
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank Microverse Family and all my coding partners.
```
- [ ] According to the requirements:

![image](https://github.com/s2jack/codewizards.gihhub.io/assets/77069011/9eae8070-a2a4-4f3d-b01b-8aefe1e83cf9)

Please consider giving the author of the design the appropriate credit! 🤓 🔧

anagudelogu commented on index.html:
```diff
@@ -0,0 +1,141 @@
+        <div id="enroll-button" class="lato red">Enroll Now</div>
+        <i id="hamburger-icon" class="fa-solid fa-bars"></i>
+      </navbar>
+    </header>
```
You did a great job working on this milestone! 💯 However, there's still some room to improve:

- [x] Please consider using the appropriate space between links in your desktop version: 🔧 

| Template |
|---|
| ![image](https://github.com/s2jack/codewizards.gihhub.io/assets/77069011/c05b73df-658f-4443-98ce-b3881e5e242a) |

| Your Design |
|---|
| ![image](https://github.com/s2jack/codewizards.gihhub.io/assets/77069011/88667d0b-cf3d-47e0-83b6-84c174212c04) |

- [x] For the mobile version, neither the logo nor the dark toolbar is present. Please consider following the design as closely as possible! 🔧 

|Template|Your design|
|---|---|
| ![image](https://github.com/s2jack/codewizards.gihhub.io/assets/77069011/f7787814-ddf8-4b48-a8c8-22990d22a25e) | ![image](https://github.com/s2jack/codewizards.gihhub.io/assets/77069011/b1208d11-06f4-4b15-8a69-742aa06c3663) |

- [x] The `<navbar>` tag isn't valid HTML. Maybe you tried to spell `<nav>`? Please consider applying this change! 🔧

- [x] I noticed that you're using the CSS `background-image` property to render your logo. Please avoid using it for content that actually contributes to the content of the page. As a rule of thumb, only use `background-image` for decorative purposes! 🔧 

- [x] I think it would be better to use a `<button>` tag instead of a `<div>` to represent your Enrollment button. Semantic tags ensure your page is accessible to people who rely on assistive technologies such as screen readers! 🤓 🔧

anagudelogu commented on index.html:
```diff
@@ -0,0 +1,141 @@
+          <button id="join-btn" class="red-bg lato white">Join CWA Full-Stack Program</button>
+          <a id="program-link" class="lato white">SEE THE WHOLE PROGRAM</a>
+        </div>
+      </section>
```
For your program section:

- [x] Please consider fixing the mobile styles, currently, the CTA button is overlapping the instructor's title! 🔧 

|Template|Your design|
|---|---|
| ![image](https://github.com/s2jack/codewizards.gihhub.io/assets/77069011/45f0c2ae-54db-4977-93a9-965556832176) | ![image](https://github.com/s2jack/codewizards.gihhub.io/assets/77069011/37f5bd00-8491-4a48-afae-8ab4c08d9321) |

- [x] I think that using the `<article>` tag instead of the `<div>` one as the card container would be the best option to semantically describe what those elements represent. The `<article>` tag is used to represent self-contained compositions, such as cards! 🔧  

- [x] Also, I think it would be better to use an `<hx>` tag for each card title. For example, `<h3>`. 🔧

anagudelogu commented on index.html:
```diff
@@ -0,0 +1,141 @@
+          <a id="program-link" class="lato white">SEE THE WHOLE PROGRAM</a>
+        </div>
+      </section>
+      <section id="speakers">
+      </section>
```
- [x] For your speakers' section, please consider removing the border lines in your mobile design: 🔧 

|Template|Your design|
|---|---|
| ![image](https://github.com/s2jack/codewizards.gihhub.io/assets/77069011/1e6ddea8-e72f-465e-b83b-d76f16429271) | ![image](https://github.com/s2jack/codewizards.gihhub.io/assets/77069011/7b23dd8a-e6f4-4fe9-9048-6fbc8f5d5aa2) |

anagudelogu commented on index.html:
```diff
@@ -0,0 +1,141 @@
+            <div div id="logo1" class="partner-logo"></div>
+            <div id="logo1" class="partner-logo"></div>
+          </div>
+        </div>
```
- [x] Please consider using `<img>` tags instead of `<div>` + `background-image` CSS. This is because these images actually hold content for your page! 🔧

anagudelogu commented on index.html:
```diff
@@ -0,0 +1,141 @@
+    </main>
+    <footer>
+      <div id="logo-container">
+      </div>
```
- [x] The same happens with your Logo here, an `<img>` tag would be more appropriate. 🔧

anagudelogu commented on about.html:
```diff
@@ -0,0 +1,133 @@
+      <div id="footer-logo">
+      </div>
+      <p id="copyright-txt" class="lato dark">2023 Code Wizards Academy - Some Rights Reserved</p>
+    </footer>
```
|Template|
|---|
| ![image](https://github.com/s2jack/codewizards.gihhub.io/assets/77069011/9fe7e6c4-3f9f-457a-9f43-3195de208d22) |

| Your design |
|---|
| ![image](https://github.com/s2jack/codewizards.gihhub.io/assets/77069011/93060267-43bc-436c-b007-0111ee7a5607) |

- [x] According to the design, the desktop version of the About page shouldn't display a Partner section. Please consider removing it from the desktop view! 🔧 

- [x] Also, the footer should have a dark background on desktop devices! 🔧

anagudelogu commented on about-desktop-styles.css:
```diff
@@ -0,0 +1,173 @@
+  gap: 12px;
+  align-items: center;
+  justify-content: center;
+}
```
- [x] I noticed that you have multiple repeated CSS properties across your stylesheet. It would be great if you can find a way of reducing this duplication of code ♻️. Remember that not repeating pieces of code will keep your codebase clean and readable. 🤓 🛠️

From [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):
![image](https://user-images.githubusercontent.com/77069011/200000947-9e58c4d8-fed4-45f8-a822-7828c6e713f7.png)

In this case, you could do this with flex:

```CSS
.flex {
  display: flex;
}
```

And then, use it in your HTML elements where you need a flex container:

```HTML
<div class="flex"> This is a flex container </div>
```

Please, consider fixing this! 🤓

anagudelogu commented on main.js:
```diff
@@ -0,0 +1,41 @@
+  closeIcon.style = 'display: block';
+  menuList.style = 'display: flex; flex-direction: column; gap: 14px; padding: 24px 0 0 16px;';
+
+  Array.prototype.forEach.call(menuElements, (element) => {
+    element.classList.add('lato');
+    element.classList.add('white');
+    element.style = 'font-size: 32px';
+  });
+
+  Array.prototype.forEach.call(menuElements, (element) => {
+    element.addEventListener('click', closeMobileMenu);
+  });
```
- [x] **[OPTIONAL]** Instead of using `Array.prototype.forEach.call` you can use the `forEach` method directly on the collection! For example:

 ```suggestion
  menuElements.forEach((element) => {
    element.classList.add('lato');
    element.classList.add('white');
    element.style = 'font-size: 32px';
  });

  menuElements.forEach((element) => {
    element.addEventListener('click', closeMobileMenu);
  });
```

And, since both loops are iterating through the same collection, you can merge them:

```suggestion
  menuElements.forEach((element) => {
    element.classList.add('lato');
    element.classList.add('white');
    element.style = 'font-size: 32px';
    element.addEventListener('click', closeMobileMenu);
  });



```

Consider applying this suggestion! 🤓 🔧

anagudelogu reviewed (changes requested):
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @s2jack, 
I'm Andres.


While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 

<img src="https://media2.giphy.com/media/3ohhwi25ISXC7Z4tMs/giphy-downsized-medium.gif"/>

## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

s2jack committed:
Fix issues

Check README links
Give appropriate credit for the author of design
Hide appbar and logo in the header for mobile versions
Change navbar element into nav
Change all images container div elements into img element
Change enroll button element into button
Fix main program padding issues
Change div into article in the program section
Remove lines in the speakers section for mobile versions

s2jack committed:
Merge branch 'main' into desktop-versions

s2jack committed:
Merge branch 'desktop-versions' of https://github.com/s2jack/codewizards.github.io into desktop-versions

s2jack committed:
Fix stylelints

AlphaNtihinduka commented on instructor-dynamic.js:
```diff
@@ -0,0 +1,104 @@
+    const order = i + 1;
+    const instructorCard = document.createElement('div');
+    instructorCard.id = `speaker-card${order}`;
+    instructorCard.className = 'speaker-card';
```
- [x] As requested by the previous reviewer, Kindly ensure that your remove the border. This is to ensure that your design matches the original design and hence making your website professional ☺️ 

Your design 😎 
![image](https://github.com/s2jack/codewizards.github.io/assets/99394374/9232887c-c20c-4d86-bf89-70174a12a19f)

Original design 😎 
![249321065-1e6ddea8-e72f-465e-b83b-d76f16429271](https://github.com/s2jack/codewizards.github.io/assets/99394374/7ddab2b5-7d0b-459a-8977-bc9cc4e94c49)

AlphaNtihinduka commented on index.html:
```diff
@@ -0,0 +1,141 @@
+            </article>
+          </div>
+          <button id="join-btn" class="red-bg lato white">Join CWA Full-Stack Program</button>
+          <a id="program-link" class="lato white">SEE THE WHOLE PROGRAM</a>
```
- [x] [OPTIONAL] I have noticed that the `see the whole program` and `instructors` overlable into the article cards. This makes your website messy and therefore poor UI. Kindly consider fixing this in all the desktop screen sizes to ensure that your application is professional ☺️.
- [x] In addition, the article cards should be vertically centered in the container and shouldn't overlap to other section of the application ☺️.

Your design 😎 
![image](https://github.com/s2jack/codewizards.github.io/assets/99394374/4495e15c-f12d-4a72-ad4e-0dfe8254f9ff)

Original design 😎 
![image](https://github.com/s2jack/codewizards.github.io/assets/99394374/14d996af-9801-4384-ba44-976143f220a8)

AlphaNtihinduka reviewed (changes requested):
## Hi @s2jack 🙋‍♂️
Good job working on implementing the changes required by the previous reviewer. You have done the design to do the expectation and incredibly well. 👏  

TO HIGHLIGHT:
- Your project is professional ✅
- Professional README file :100: 
- Your PR is not yet professional ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

![close](https://user-images.githubusercontent.com/99394374/210771148-619a4265-33b2-4e68-8901-6bb52693d6de.gif)

## Required changes :recycle: 
- Kindly consider the inline review commits ☺️.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

s2jack committed:
Fix issues

Add min-height property to the program section
Delete border of the instructors cards

AbelG101 reviewed (approved):
![](https://media.giphy.com/media/eKfovWZOlOpbTDEX9F/giphy.gif)

# Hi @s2jack 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- Content is uniquely tailored ✔
- UI follow original design ✔
- Well-structured files ✔
- Detailed PR summary ✔

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
